src/api/spec/factories/bs_requests.rb

```diff
@@ -24,5 +28,46 @@
     end
 
     after(:create) { User.current = nil }
+
+    factory :bs_request_with_submit_action do
+      after(:create) do |request, evaluator|
+        request.bs_request_actions.delete_all
```

Comment:
there is not other way to do this? it is a little bit ugly to create always an action and then delete it here.